provides.py

```diff
@@ -38,14 +38,23 @@ def joined_or_changed(self):
         if self._has_auth_request():
             conv.set_state('{relation_name}.auth.requested')
 
-    @hook('{provides:kube-control}-relation-{broken,departed}')
+    @hook('{provides:kube-control}-relation-departed')
     def departed(self):
         """Remove all states.
 
         """
         conv = self.conversation()
         conv.remove_state('{relation_name}.connected')
         conv.remove_state('{relation_name}.gpu.available')
+        conv.set_state('{relation_name}.departed')
```

Comment:
Is there a meaningful distinction between `kube-control.departed` and `not kube-control.connected`?

I guess it's so we can get the unit that's departing? That seems reasonable I suppose.